week-2-practise/level-2/2-validation.js

```diff
@@ -1,11 +1,30 @@
+import { numberLiteralTypeAnnotation, returnStatement } from "@babel/types";
+
 // Complete the function to check if the variable `num` satisfies the following requirements:
 // - is a number
 // - is a positive number
 // - is less than or equal to 100
 // Tip: write other small functions for each requirement
 
-function validate(num) {}
+function validate(num) {
+  return isNumber(num) && isPositive(num) && isLessThanOrEqual(num);
+}
+
+function isNumber(number) {
+  return typeof number === "number";
+}
+
+function isPositive(number) {
+  // var psotiveNumer = number > 0;
+  // return psotiveNumer ;
+  return number >= 0;
+}
 
+function isLessThanOrEqual(number) {
```

Comment:
You might be confused by this function name if you come back to it in a few months! You might want to put oneHundred in the name (or maybe it really is easier to just have `num <= 100` on line 10).